cli/command/run/cmd.go

```diff
@@ -3,6 +3,7 @@ package run
 import (
 	"errors"
 	"fmt"
+	"github.com/saucelabs/saucectl/internal/docker"
```

Comment:
Maybe this one should be move to the other group of imports ?

Reply:
Not sure I follow?

Reply:
Move it to imports starting with `github.com/saucelabs/saucectl/` (15 lines below)